src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -27,5 +27,11 @@
   <ItemGroup>
     <ReferenceAssemblyProjectReference Include="..\Microsoft.Identity.Client.Ref\Microsoft.Identity.Client.Ref.csproj" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Please undo the changes to this file. Visual Studio gets a little confused sometimes when adding files, but the general pattern is that we include all .cs files by default, so this is not needed.